python/ruff-ecosystem/ruff_ecosystem/cli.py

```diff
@@ -121,7 +121,7 @@ def parse_args() -> argparse.Namespace:
     parser.add_argument(
         "--output-format",
         choices=[option.name for option in OutputFormat],
-        default="json",
+        default="markdown",
```

Comment:
Do we need to pass this in somewhere now?

Reply:
No in CI we pass `--output-format markdown` already